relocation/types.go

```diff
@@ -0,0 +1,4 @@
+package relocation
+
+// ImageRelocationMap stores the mapping between the original service image as key, and the relocated reference as a value.
```

Comment:
Why assume all images are services?
```suggestion
// ImageRelocationMap stores the mapping between the original image reference as key, and the relocated reference as a value.
```

Reply:
Right 👍